indico_zodbimport/modules/event_regforms.py

```diff
@@ -151,6 +153,9 @@ def _migrate_settings(self):
         self.regform.manager_notification_recipients = sorted(set(old_rf.notification._ccList) |
                                                               set(old_rf.notification._toList))
         self.regform.manager_notifications_enabled = bool(self.regform.manager_notification_recipients)
+        if (layout_settings.get(self.event, 'use_custom_menu')
```

Comment:
Also, this sends an extra query for each registration.  I'd load the relevant information in `EventRegformImporter.load_data`. since we never keep menu data for events where customization is disabled in the db, you can simply do a query like this

    self.participant_list_disabled = {x for x, in db.session.query(MenuEntry.event_id).filter(MenuEntry.name == 'participants', ~MenuEntry.is_enabled)}

And here you simply use `self.importer.participant_list_disabled.get(int(self.event.id), False)`